libs/protocol-ui/src/components/molecules/ControlledSelect/ControlledSelect.tsx

```diff
@@ -24,9 +24,9 @@ export interface ControlledSelectProps {
   isMulti?: boolean;
   isClearable?: boolean;
   onChange?: (option: { value: number; label: string }) => void;
-  isDisabled?: boolean;
   variant?: 'outline' | 'filled';
   value?: any;
```

Comment:
This is lingering from our type fixes -- I removed about 30 warnings and cleaned the UI package up considerably, but this is on my list